app/api/users.py

```diff
@@ -244,23 +240,22 @@ def before_get_object(self, view_kwargs):
 
         if view_kwargs.get('access_code_id') is not None:
             access_code = safe_query(
-                self, AccessCode, 'id', view_kwargs['access_code_id'], 'access_code_id'
+                AccessCode, 'id', view_kwargs['access_code_id'], 'access_code_id'
             )
             if access_code.marketer_id is not None:
                 view_kwargs['id'] = access_code.marketer_id
             else:
                 view_kwargs['id'] = None
 
         if view_kwargs.get('event_id') is not None:
-            event = safe_query(self, Event, 'id', view_kwargs['event_id'], 'event_id')
+            event = safe_query(Event, 'id', view_kwargs['event_id'], 'event_id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)